packages/app-desktop/gui/NoteEditor/utils/useMarkupToHtml.ts

```diff
@@ -40,6 +42,7 @@ export default function useMarkupToHtml(deps: HookDependencies) {
 		options = {
 			replaceResourceInternalToExternalLinks: false,
 			resourceInfos: {},
+			platformName: shim.platformName(),
```

Comment:
I don't understand how you managed to pass platformName here, because RuleOptions doesn't seem to have this property?